inductiva/resources/machines_base.py

```diff
@@ -0,0 +1,214 @@
+"""Base class for machine groups."""
+import time
+
+from absl import logging
+
+import inductiva
+import inductiva.client.models
+from inductiva import api
+from inductiva.client.apis.tags import instance_api
+
+
+class BaseMachineGroup():
+    """Base class to manage Google Cloud resources."""
+    def __init__(
+        self,
+        machine_type: str,
+        spot: bool = False,
+        disk_size_gb: int = 40,
+        zone: str = "europe-west1-b"
+    ) -> None:
+        """Create a MachineGroup object.
```

Comment:
The value of 40Gb for disk_size_gb should not be hardcoded here because it is a really important value for the business.
Can we make it a constant for now, defined somwhere else, so we can edit it a propagate to all cases?

@hpenedones : wdyt? Should we move all these business oriented values to an "external" file?

Reply:
To be honest, that constant value should be in the backend... Here, it should be an argument that the user can choose that just refers to the amount of disk available for his simulation data (before it goes into the bucket). 

Remark that by adding a new simulator or docker image, this value will need to increase and the user will have to change it without any awareness of what happened... It is a current flaw and should be refactor to just refer to the disk available to the user.

Reply:
Yes, it should be in the backend. But even this one should be in just a single place in the client.
I think there are at least two places in the client where we mention this value. As a first step, there should be a const that is the default value for both cases.